src/com/etendoerp/metadata/BaseServlet.java

```diff
@@ -43,12 +43,17 @@ public final void service(HttpServletRequest request, HttpServletResponse respon
         }
     }
 
-    protected DecodedJWT getDecodedToken(HttpServletRequest request) throws Exception {
-        String token = getToken(request);
-        DecodedJWT decodedToken = decodeToken(token);
+    protected DecodedJWT getDecodedToken(HttpServletRequest request) {
+        try {
+            String token = getToken(request);
+            DecodedJWT decodedToken = decodeToken(token);
 
-        if (decodedToken != null) return decodedToken;
-        else throw new UnauthorizedException();
+            if (decodedToken != null) return decodedToken;
```

Comment:
I prefer to have the curly braces even if it's a one-line if/else expression, but besides that, LGTM

Reply:
I just wanted to save one line 😅
Jokes aside, you're right. 
I agree, in terms of readability is better to always keep the curly braces.
Thanks for your feedback!

Reply:
It's due to a Java standard convention: https://www.oracle.com/java/technologies/javase/codeconventions-statements.html#449

Reply:
Shame on me, I didn't know. 
I'll fix all the bad practices I've applied and update the PR 

Many thanks for your help!